src/currencyservice/server.js

```diff
@@ -116,16 +138,28 @@ function _carry (amount) {
  * Lists the supported currencies
  */
 function getSupportedCurrencies (call, callback) {
+  const currentSpan = tracer.getCurrentSpan();
+  const span = tracer.startSpan('currencyservice:GetSupportedCurrencies()', {
+    parent: currentSpan,
+    kind: 1, //server
```

Comment:
why it's server?

Reply:
Currency Service is the server and whoever is calling this function (i.e. frontend or checkout) is the client